gopsutil/systeminfo.go

```diff
@@ -40,6 +46,55 @@ func (s *systemInfo) Uptime() (uptime uint64, err error) {
 	return hostInfo.Uptime, nil
 }
 
+// cpuFrequencyMultipliers is a lookup table for prefixes on clock speeds.
+// This is probably overkill.
+var cpuFrequencyMultipliers = [256]int{
+	'M': 1,
+	'G': 1000,
+	'T': 1000 * 1000,
+}
```

Comment:
I did not know you could do this... not sure how I feel about it.

Reply:
This is two syntactic sugar at once. One is that you can use labeled initializers in slice/array constants, just like in struct ones, the other is that you can treat runes as their unicode values.

Very convenient, though.